moveit_core/distance_field/include/moveit/distance_field/voxel_grid.h

```diff
@@ -61,6 +62,8 @@ template <typename T>
 class VoxelGrid
 {
 public:
+  MOVEIT_DECLARE_PTR_MEMBER(VoxelGrid);
+
```

Comment:
I don't really see a reason why we should not have just call the member type `VoxelGridPtr` instead of `Ptr`, but then I suppose it does not really matter.